infra-library.sh

```diff
@@ -137,5 +137,11 @@ function create_gke_test_cluster() {
   local -n _custom_flags=$1
   local -n _test_command=$2
 
-  run_kntest kubetest2 gke "${_custom_flags[@]}" --test-command="${_test_command[*]}"
+# We are disabling logs and metrics on Boskos Clusters by default as they are not used. Manually set GKE_DISABLE_TELEMETRY to false to enable logging
```

Comment:
Super nit: the indentation is not correct.

Reply:
fixed